saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -71,6 +90,188 @@ class Meta:
         error_type_field = "product_errors"
 
 
+class ProductVariantBulkCreateInput(ProductVariantInput):
+    attributes = graphene.List(
+        AttributeValueInput,
+        required=True,
+        description="List of attributes specific to this variant.",
+    )
+
+
+class ProductVariantBulkCreate(BaseMutation):
+    count = graphene.Int(
+        required=True,
+        default_value=0,
+        description="Returns how many objects were affected.",
+    )
+    product_variants = graphene.List(
+        ProductVariant,
```

Comment:
This should be wrapped in `graphene.NonNull` for better typing in the schema. We'll never get a list with `None` values inside.